common/app/views/fragments/meta/metaInline.scala.html

```diff
@@ -32,7 +32,7 @@
     }
 
     @item.content.blogOrSeriesTag.map { series =>
-    <div class="content__series-label @if(item.content.isImmersive && item.content.tags.isArticle) { content__series-label--immersive-article }">
+    <div class="content__series-label @if(item.content.isImmersive && item.content.tags.isArticle) { content__series-label--immersive-article } @if(item.content.isPhotoEssay) { content__series-label--photo-essay}">
```

Comment:
Do you think we could use [Renderclasses](https://github.com/guardian/frontend/blob/master/common/app/views/main.scala.html#L40) instead of if statements here? It will just make it a bit more readable.